tests/integrationv2/test_session_resumption.py

```diff
@@ -115,6 +115,9 @@ def test_tls13_session_resumption_s2n_server(managed_process, tmp_path, cipher,
     p = tmp_path / 'ticket.pem'
     path_to_ticket = str(p)
 
+    server_send_marker = 's2n is ready'
+    data = data_bytes(10)
```

Comment:
You will need to rebase to include the framework fixes from this commit: https://github.com/aws/s2n-tls/commit/98aa22805fab242c7e268a79a0b0672d2bccd793.